src/Http/Middleware/ForbidBannedUser.php

```diff
@@ -45,11 +45,17 @@ public function __construct(Guard $auth)
     public function handle($request, Closure $next)
     {
         $user = $this->auth->user();
+        $redirectUrl = config('ban.redirect_url', null);
```

Comment:
It would be better to move this variables inside of the `if` block, because it's using only there.

Reply:
Moved vars inside if block